common4j/src/main/com/microsoft/identity/common/java/logging/RequestContextGuard.java

```diff
@@ -0,0 +1,27 @@
+package com.microsoft.identity.common.java.logging;
+
+import com.microsoft.identity.common.java.util.StringUtil;
+
+public class RequestContextGuard implements AutoCloseable {
```

Comment:
This class seems to not do anything special. Why do we need this class? Why can't we do what it does directly in the DiagnosticContext class?